openmsupply/open-msupply
# 2103 contact tracing various

Part of #2103 

- Rename contact trace tab to "Contact Tracing"
- Removes contact tracing status (needs more discussion and is not needed right now)
- Remove status from UI
- Add first and last name
- Add dob and gender
- Remove full contact tracing list view
- Add store_id for each contact trace

# Testing:
- Check contact tracing detail view and list view
- Check that a contact trace DB entry has the current store id
